### PR TITLE
fix(probit) - WS reorganize

### PR DESCRIPTION
**Pull request by ttodua**
issues reported here :https://discord.com/channels/690203284119617602/690547932663119902/1311117428079464492

I've manually tested all public & private methods after update, they are now fixed.
beside impementation/method updates, now it's possible to subscribe to multiple channels for same symbol without unsubscribing from previous

### CODE REVIEW

File: ts/src/pro/probit.ts [hunk 1]
```diff
@@ -314,6 +300,10 @@ export default class probit extends probitRest {
         client.resolve (stored, messageHash);
     }
 
+    parseTrade (rawTrade: Dict, market: Market = undefined): Trade {
+        return super.parseTrade (rawTrade, market);
+    }
+
```
Comment (carlosmiei) [inline ts/src/pro/probit.ts:306]: why this? why don't we just call parseTrade then?

File: ts/src/pro/probit.ts [hunk 2]
```diff
@@ -36,15 +36,7 @@ export default class probit extends probitRest {
...
-                'watchTrades': {
-                    'filter': 'recent_trades',
-                },
-                'watchTicker': {
-                    'filter': 'ticker',
-                },
-                'watchOrders': {
-                    'channel': 'open_order',
-                },
+                'tempFiltersData': {}, // temporary container
```
Comment (carlosmiei) [inline ts/src/pro/probit.ts:39]: is this being used?

File: ts/src/pro/probit.ts (hunk 1 above)
Comment (ttodua) [reply ts/src/pro/probit.ts:306]: I was planning to add some additional parsing in `ws` parsing here, but havent added. maybe yes, i should remove these lines now.

File: ts/src/pro/probit.ts (hunk 2 above)
Comment (ttodua) [reply ts/src/pro/probit.ts]: @carlosmiei right, remnant. I've pushed all changes per your notes

**Comment by carlosmiei** [comment]
tested and seems to be working as expected
```
CCXT Version: 4.4.34
Probit:watchTrades
BTC/USDT
ETH/USDT
BTC/USDT
ETH/USDT
ETH/USDT
```

**Pull request merged**